install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -3,20 +3,37 @@
 ----
 [OSEv3:vars]
 ...
-openshift_registry_selector="role=infra" <1>
+openshift_registry_selector="role=infra"                  <1>
 openshift_hosted_registry_storage_kind=glusterfs
 
+openshift_metrics_hawkular_nodeselector={"role":"infra"}  <1>
+openshift_metrics_cassandra_nodeselector={"role":"infra"} <1>
+openshift_metrics_heapster_nodeselector={"role":"infra"}  <1>
+openshift_metrics_storage_kind=dynamic
+openshift_metrics_cassanda_pvc_storage_class_name="glusterfs-registry-block" <2>
+
+openshift_logging_es_nodeselector={"role":"infra"}        <1>
+openshift_logging_kibana_nodeselector={"role":"infra"}    <1>
+openshift_logging_curator_nodeselector={"role":"infra"}   <1>
+openshift_logging_storage_kind=dynamic
+openshift_logging_es_pvc_size=10Gi                        <3>
+openshift_logging_es_pvc_storage_class_name="glusterfs-registry-block"       <2>
+
 openshift_storage_glusterfs_block_deploy=false
 
 openshift_storage_glusterfs_registry_block_deploy=true
 openshift_storage_glusterfs_registry_block_storageclass=true
-openshift_storage_glusterfs_registry_block_storageclass_default=true
-
-openshift_storageclass_default=false
-----
-<1> It is recommended to run the integrated OpenShift Container Registry on
-nodes dedicated to "infrastructure" applications, that is applications deployed
-by administrators to provide services for the {product-title} cluster.
+openshift_storage_glusterfs_registry_block_storageclass_default=false
+----
+<1> It is recommended to run the integrated OpenShift Container Registry,
+Logging, and Metrics on nodes dedicated to "infrastructure" applications,
+that is applications deployed by administrators to provide services for the
+{product-title} cluster.
+<2> Specify the StorageClass to be used for Logging and Metrics. This name is
+generated from the name of the target GlusterFS cluster (e.g.,
```

Comment:
s/ (e.g., `glusterfs-<name>-block`)./for example `glusterfs-<name>-block`.

Reply:
Is `e.g.` not allowed for some reason?

Reply:
Yes, it's not allowed by our style guide.

Reply:
Good to know!

Reply:
Dangit. :) Will fix as well.